batchglm/train/numpy/glm_norm/utils.py

```diff
@@ -0,0 +1,7 @@
+import math
+import numpy as np
+
+def ll(scale, loc, x):
```

Comment:
shouldn't this be in `model_container`?

Reply:
I guess but it's not a class method.  I could move it there though.

Reply:
we should make it consistent though. Is this resolved?

Reply:
yes it's in that now.